testing/utilities.py

```diff
@@ -416,6 +418,18 @@ class partialTestAPI(partialChecks):
 
     '''
 
+    def test_get_version(self):
+        '''Test getting the version of BOPTEST.
+
+        '''
+
+        version = requests.get('{0}/version'.format(self.url)).json()
+        r = re.compile('\d.\d.\d')
```

Comment:
maybe add some comments here for maintenance purposes

Reply:
Done in https://github.com/ibpsa/project1-boptest/pull/336/commits/9955ba4fcfbf3596bb050a56d3ca90e9508a4556.